dashboard/test/models/user_test.rb

```diff
@@ -1563,6 +1563,18 @@ def track_progress(user_id, script_level, result, pairings: nil)
     assert_equal [], teacher.reload.permissions
   end
 
+  test 'assign_course_as_facilitator assigns course to facilitator' do
+    facilitator = create :facilitator
+    facilitator.course_as_facilitator = Pd::Workshop::COURSES.last
```

Comment:
Thanks for explaining how `assert_creates` and `assert_difference` work.

The change you suggested works:
``` ruby
    assert_creates Pd::CourseFacilitator do
      facilitator.course_as_facilitator = Pd::Workshop::COURSE_CS_IN_A
    end
    course_facilitator = Pd::CourseFacilitator.last
    assert_equal Pd::Workshop::COURSE_CS_IN_A, course_facilitator.course
    assert_equal facilitator, course_facilitator.facilitator 
```

But it appears to be more verbose and less direct than:
``` ruby
facilitator.course_as_facilitator = Pd::Workshop::COURSE_CS_IN_A
assert facilitator.courses_as_facilitator.where(:course Pd::Workshop::COURSE_CS_IN_A).any?
```

Also in real-life usage, `course_as_facilitator=` does not necessarily create a record that is Pd::CourseFacilitator.last if that course was already assigned to the facilitator.